src/test/java/org/mamute/auth/rules/EditAnswerRuleTest.java

```diff
@@ -24,29 +30,39 @@ public void setUp(){
 	}
 
 	@Test
-	public void author_should_be_allowed_to_edit_moderatable() {
-		assertTrue(new EditAnswerRule(author).isAllowed(answer));
+	public void author_should_be_allowed_to_edit_moderatable() throws IOException {
+		ServletContext ctx = mock(ServletContext.class);
+		EnvironmentKarma env = new EnvironmentKarma(new MamuteEnvironment(ctx));
+		assertTrue(new EditAnswerRule(author, env).isAllowed(answer));
 	}
 	
 	@Test
-	public void loggedUser_with_enough_karma_should_be_allowed_to_edit() {
+	public void loggedUser_with_enough_karma_should_be_allowed_to_edit() throws IOException {
```

Comment:
Does MamuteEnvironment throws IOException? Maybe its better to catch it inside its constructor.